benchmarks/cuqi-ct/test_output.py

```diff
@@ -0,0 +1,159 @@
+parser.add_argument('url', metavar='url', type=str,
+                    help='the ULR on which the model is running, for example http://localhost:4242')
+args = parser.parse_args()
+print(f"Connecting to host URL {args.url}")
+
+# Define models for each type of prior
+print('Setting up models')
+model_Gaussian = umbridge.HTTPModel(args.url, "CT_Gaussian")
+model_GMRF = umbridge.HTTPModel(args.url, "CT_GMRF")
+model_CMRF = umbridge.HTTPModel(args.url, "CT_CMRF")
+model_LMRF = umbridge.HTTPModel(args.url, "CT_LMRF")
+
+# Model for exact solution
+model_exactSolution = umbridge.HTTPModel(args.url, "CT_ExactSolution")
+
+# Check basic model properties
+print('Checking model properties')
+assert model_Gaussian.get_input_sizes() == [256**2]
+assert model_GMRF.get_input_sizes() == [256**2]
+assert model_CMRF.get_input_sizes() == [256**2]
+assert model_LMRF.get_input_sizes() == [256**2]
+assert model_exactSolution.get_input_sizes() == [0]
+
+assert model_Gaussian.get_output_sizes() == [1]
+assert model_GMRF.get_output_sizes() == [1]
+assert model_CMRF.get_output_sizes() == [1]
+assert model_LMRF.get_output_sizes() == [1]
+assert model_exactSolution.get_output_sizes() == [256**2]
+
+# Check that the models support evaluation and gradient
+print('Checking model capabilities')
+assert model_Gaussian.supports_evaluate()
+assert model_GMRF.supports_evaluate()
+assert model_CMRF.supports_evaluate()
+assert model_LMRF.supports_evaluate()
+assert model_exactSolution.supports_evaluate()
+
+assert model_Gaussian.supports_gradient()
+assert model_GMRF.supports_gradient()
+assert model_CMRF.supports_gradient()
+assert not model_LMRF.supports_gradient()
+assert not model_exactSolution.supports_gradient()
+
+# Evaluate models and compare with testproblem
+print('Evaluating models')
+output_Gaussian = model_Gaussian([parameters.tolist()])[0][0]
+output_GMRF = model_GMRF([parameters.tolist()])[0][0]
+output_CMRF = model_CMRF([parameters.tolist()])[0][0]
+output_LMRF = model_LMRF([parameters.tolist()])[0][0]
+output_exactSolution = model_exactSolution([[]])[0]
+
+# Check that the output is correct
+print('Checking model output')
+BP.prior = cuqi.distribution.Gaussian(np.zeros(256**2), 0.01,
+                              geometry=BP.likelihood.geometry)
+assert output_Gaussian == pytest.approx(BP.posterior.logpdf(parameters))
+
+BP.prior = cuqi.distribution.GMRF(np.zeros(256**2), 1/(0.01),
+                                  physical_dim=2,
+                                  geometry=BP.likelihood.geometry)
+assert output_GMRF == pytest.approx(BP.posterior.logpdf(parameters))
+
+BP.prior = cuqi.distribution.Cauchy_diff(np.zeros(256**2), 0.01,
+                                         physical_dim=2,
+                                         geometry=BP.likelihood.geometry)
+assert output_CMRF == pytest.approx(BP.posterior.logpdf(parameters))
+
+BP.prior = cuqi.distribution.Laplace_diff(np.zeros(256**2), 0.1,
```

Comment:
failed the test. Wrong parameter
```suggestion
BP.prior = cuqi.distribution.Laplace_diff(np.zeros(256**2), 0.01,
```